pennylane/_device.py

```diff
@@ -340,6 +339,7 @@ def check_validity(self, queue, observables):
                 if o.name not in self.observables:
                     raise DeviceError("Observable {} not supported on device {}".format(o.name, self.short_name))
 
+    @property
```

Comment:
This (as well as apply, expval, reset) are not properties :slightly_smiling_face: 
```suggestion
```

Reply:
Great, thanks for that!!